cmd/yamlfmt/main.go

```diff
@@ -34,6 +38,15 @@ func run() error {
 	configureHelp()
 	flag.Parse()
 
+	if *flagVersion {
+		_, err := fmt.Printf("%s\n", version)
+		if err != nil {
+			return err
+		}
+
+		os.Exit(0)
```

Comment:
This can just `return nil` here; it exits the program the same way.
```suggestion
		return nil
```

Reply:
Thank you! I'm learning golang again from this project... 🙏 

https://github.com/google/yamlfmt/pull/117/commits/f6168428420b5da0d765a053cf9ca8a15347ac03

Reply:
I appreciate you making the effort to contribute in something that isn't your main language! Means a lot :smile: